### PR TITLE
CONCF-477 Fixes background logic for breakpoints for AdSkin 1064

### PR DESCRIPTION
**Pull request by vforge**
Ping @bogna and @hakubo  and @gabrys

### CODE REVIEW

**Comment by gabrys** [comment]
The part when you pass ten64 to BackgroundChanger LGTM

File: skins/oasis/css/core/breakpoints-background.scss [hunk 1]
```diff
@@ -25,16 +19,15 @@ $large-breakpoint-content-and-adskin-width: $content-width-large + 2 * $adskin-w
...
 	background: rgba($color-body-middle, 0) linear-gradient(to #{$side}, rgba($color-body-middle, 0) 0%, $color-body-middle 100%);
 }
 
 @mixin background-max() {
-	$background-margin: ($content-width-medium / 2) - (($content-width-large / 2) + $background-width-half + $legacy-background-shift);
-	$background-halves-spacing: $content-width-large - $content-width-medium + $legacy-background-shift-both-sides;
-	$gradient-offset: ($background-halves-spacing / 2) + $gradient-width;
+	$background-margin: ($content-width-medium / 2) - (($content-width-large / 2) + $background-width-half);
+	$background-halves-spacing: $content-width-large - $content-width-medium;
+	$gradient-width: ($background-halves-spacing / 2) + $gradient-width;
```
Comment (bognix) [inline skins/oasis/css/core/breakpoints-background.scss]: shouldn't it be gradient-offset? gradient should always be 40px wide.

File: skins/oasis/css/core/breakpoints-background.scss [hunk 2]
```diff
@@ -110,28 +103,33 @@ body.skin-oasis {
...
 		.background-image-gradient {
 			background: $color-body-middle;
+			content: '';
+			height: 100%;
 			/* @noflip */
 			left: 50%;
 			/* @noflip */
-			margin-left: -#{$legacy-background-shift};
-			width: $legacy-background-shift-both-sides;
+			margin-left: 0;
```
Comment (bognix) [inline skins/oasis/css/core/breakpoints-background.scss]: does it have to be 0? maybe we don't need to set it.

File: skins/oasis/css/core/breakpoints-background.scss [hunk 3]
```diff
@@ -110,28 +103,33 @@ body.skin-oasis {
...
+			position: fixed;
+			top: 0;
+		}
 
-			&,
+		.background-image-gradient:not(.no-gradients) {
```
Comment (bognix) [inline skins/oasis/css/core/breakpoints-background.scss]: would simple check like:
```
&.no-gradients {
    display:none;
}
wouldn't work?

**Comment by bognix** [comment]
LGTM

**Comment by vforge** [comment]
Not more needed. Replaced with CONCF-477-525 branch.

Closing this one. Thanks for the ride.

**Pull request closed without merging**